src/get_data.py

```diff
@@ -1,12 +1,12 @@
 import requests
 import json
 
-QURERY_PARAMS = 'species:mouse&fq=cell_type:pyramidal&fq=brain_region:neocortex'
+QURERY_PARAMS = 'species:mouse,rat&fq=cell_type:pyramidal&fq=brain_region:neocortex'
 
 def get_swc(archive_name, CNG_version, neruon_name, neuron_id):
     url = 'http://neuromorpho.org/dableFiles/' + archive_name.lower() + '/' + CNG_version + '/' + neruon_name + '.CNG.swc'
     response = requests.get(url)
-    with open(r'./swc_files/' + str(neuron_id) + '.CNG.swc', 'wb') as f:
+    with open(r'../swc_files/' + str(neuron_id) + '.CNG.swc', 'wb') as f:
```

Comment:
Please use fstrings instead of this atrocity

Reply:
no